lib/iris/tests/unit/analysis/trajectory/test_UnstructuredNearestNeighbourRegridder.py

```diff
@@ -7,22 +7,21 @@
 
 """
 
-# Import iris.tests first so that some things can be initialised before
-# importing anything else.
-import iris.tests as tests  # isort:skip
-
 import numpy as np
+import pytest
 
 from iris.analysis.trajectory import UnstructuredNearestNeigbourRegridder as unn_gridder
 from iris.coord_systems import GeogCS, RotatedGeogCS
 from iris.coords import AuxCoord, DimCoord
 from iris.cube import Cube, CubeList
+from iris.tests._shared_utils import assert_array_equal
 
 
 class MixinExampleSetup:
     # Common code for regridder test classes.
 
-    def setUp(self):
+    @pytest.fixture(autouse=True)
+    def _setup(self):
```

Comment:
```suggestion
    def _setup_mixin(self):
```
Yours works fine, and in the current state of the tests is perfectly valid. Changing it to this might just make it more future proof?

Reply:
Agreed!